shared/chat/conversation/attachment-get-titles/index.tsx

```diff
@@ -85,7 +86,7 @@ class GetTitles extends React.Component<Props, State> {
     let preview: React.ReactNode = null
     switch (info.type) {
       case 'image':
-        preview = path ? <Kb.ZoomableImage src={path} style={styles.image} /> : null
+        preview = path ? <Kb.ZoomableImage src={info.url ?? path} style={styles.image} /> : null
```

Comment:
support kbfs 'send to chat converation' flows in previews, bug in master as well